pkg/operator/api/api.go

```diff
@@ -33,18 +33,20 @@ const (
 )
 
 type Cluster struct {
-	clientset kubernetes.Interface
-	Namespace string
-	Version   string
-	Replicas  int32
+	clientset   kubernetes.Interface
+	ClusterName string
```

Comment:
I'll look at the naming again. I remember considering `Name`, but don't recall why i didn't go with it